gpio/AM2302_sensor/AM2302-Sensor.cpp

```diff
@@ -0,0 +1,252 @@
+/*
+*	AM2302-Sensor.cpp
+*
+*	Author: Frank Häfele
```

Comment:
Could you add some licensing-information please?

Reply:
@lurch 
any prefererations?

Reply:
As you're the author, it's obviously up to you. But if you choose the same licence as all of our other examples, that's probably easiest.

Reply:
@lurch fixed, hope it is fine.

Reply:
For the avoidance of doubt, it's probably best to add the `License-Identifier` to each of the header-blocks in which you've listed yourself as the Author?